website/docs/tech-explanation/extreme-decentralization.md

```diff
@@ -0,0 +1,91 @@
+
+This philosophy dictates a clear division of labor, ensuring every point on the spectrum receives dedicated effort from the actors best suited for it:
+
+- **Layer 1 (CKB):** Focuses exclusively on being the ultimate guardian of value. Its design prioritizes maximum decentralization, security, and censorship resistance—the original spirit of crypto—above all else.
+- **Layer 2 & Ecosystem:** Focuses on scalability, high performance, the developer and user experience necessary for mass adoption, free from the burden of base-layer consensus trade-offs.
+
+It is not only about code, but also for operational decisions. For example, the CKB team might deliberately reject providing centralized conveniences, such as a default, production RPC service (similar to Infura in Ethereum). While such a service would offer a shortcut for developers, it introduces a central point of failure. Note that the [public RPC service](/docs/getting-started/rpcs) that CKB team provide is only for testing purpose, it is strongly recommended to run your own RPC service for your dApp. By refusing to encroach on the "service layer," CKB forces the ecosystem to build its own robust, distributed access layers, ensuring the foundation remains a neutral, decentralized settlement platform.
+
+## How to Achieve Extreme Decentralization
+
+Decentralization is not just about who _mines_ the next block; it is about who can _verify_ the history. If running a node becomes too expensive due to data bloat, the network centralizes into a few server farms regardless of its consensus algorithm.
+
+CKB ensures this never happens through three radical approaches: **Sustainable State Management**, **Proof-of-Work Consensus**, and a viable **Light Client** with an engineering effort on building nodes that lowers the entry barrier for participants.
+
+### 1. Sustainable State Management
+
+Most blockchains operate as "General Computation Networks" (like a world computer). Users pay a one-time fee to execute a transaction, but the resulting data (state) occupies the network's storage forever at no further cost. This leads to the **"Tragedy of the Commons"**: because storage is effectively free, the state grows indefinitely ([State Explosion](https://medium.com/nervosnetwork/state-explosion-and-the-tragedy-of-the-blockchain-commons-1fbd4837e859)). Eventually, only enterprise-grade hardware can store the chain, forcing regular users to trust third parties.
+
+CKB adopts a "General Verification Network" architecture based on [Cell Model](/docs/tech-explanation/cell-model). The **Common Knowledge Base** acts as a verified state layer rather than a computation layer. To maintain "Extreme Decentralization," CKB creates a direct economic link between state and scarcity:
```

Comment:
```suggestion
CKB, instead, adopts a "General Verification Network" architecture based on the [Cell Model](/docs/tech-explanation/cell-model). The **Common Knowledge Base** acts as a state verification layer rather than a computation layer. To maintain decentralization, CKB directly ties state scarcity with economic cost:
```